phpstan.neon.dist

```diff
@@ -18,3 +18,5 @@ parameters:
     # Probably needs Symfony plugin
     - message: '#Call to an undefined method Symfony\\Component\\Config\\Definition\\Builder\\Node#'
       path: src/DependencyInjection/Configuration.php
+    - '#Entity::\$id \(int\|null\) is never assigned int so it can be removed from the property type#'
+    - '#Entity::\$id is never read, only written#'
```

Comment:
Can we pls inline these instead? These errors are not something that's going away with some upgrade, plugin or so